### PR TITLE
resolve name inconsistency

### LINKED ISSUE #117: Create name consistency with google or gcp images
In our code we often use google or gcp for the google/gcp images. 
We should change that to `google` to be consistent.

### PR DESCRIPTION
**Pull request by miyunari**
closes #117

### CODE REVIEW

File: src/rhelocator/api/routes/google.py [hunk 1]
```diff
@@ -15,12 +15,12 @@ def gcp_blueprint(data: list[dict[str, str]]) -> Blueprint:
     Returns:
         Flask blueprint serving the provided data.
     """
-    gcp_endpoint = Blueprint("gcp_api", __name__)
+    google_endpoint = Blueprint("google_api", __name__)
 
-    @gcp_endpoint.route("/gcp", methods=["GET"])
-    @swag_from("../schema/gcp.yml")
+    @google_endpoint.route("/google", methods=["GET"])
```
Comment (miyunari) [inline src/rhelocator/api/routes/google.py:20]: @FKolwa does it make sense to rename our endpoint too? Wasn't sure about this

Comment (FKolwa) [reply src/rhelocator/api/routes/google.py:20]: Hm. I'm a bit torn on this one too. I'd like to stay as consistent as possible but gcp would probably make more sense as an endpoint.

**Review by FKolwa — APPROVED** [review]
LGTM!

**Comment by major** [comment]
Such consistency. 👏🏻

**Pull request merged**